### PR TITLE
Sprint6 fix profile page aesthetics

### PR DESCRIPTION
**Pull request by jlu51**
Updated the layout and coloring of both Viewable and Editable Profile Pages. 

Also updated the colors of the Friend Bar and Group's Bar to match the dark theme.

Removed the broken image link for now.

### CODE REVIEW

**Review by dnguyen3031 — COMMENTED** [review]
Code looks good, would suggest making the editable/viewable profile pages more readable due to the harsh looking layers of cols/row inserts, but if it's not possible then that's alright.

**Review by mchris25 — COMMENTED** [review]
Code looks great. I think some quick descriptions on these functions would be great, but not necessary as we can just ask you what they do.

File: src/components/FriendBar.js [hunk 1]
```diff
@@ -49,68 +28,53 @@ function FriendBar(props) {
...
-   }, [props._id]);
-
+   }, []);
 
-   function FriendsList(props) {
-      const [, setFriendList] = React.useState([]);
-      const [responseList, setResponseList] = React.useState([]);
+   // React.useEffect(() => {
+          
+   // }, []);
```
Comment (prcox) [inline src/components/FriendBar.js:36]: I would suggest removing old code

**Review by prcox — APPROVED** [review]
Looks good, I don't have too many comments since it was mostly astetic changes

File: src/components/pages/EditableProfilePage.js [hunk 2]
```diff
@@ -1,151 +1,179 @@
...
+               <Col xs={8} className="main-col pr-0">
+                  <Row>
+                     <Col>
+                        <Row className="pt-3 pb-3">
+                           {/* TODO: Fix pictures */}
+                           {/* <Col>
+                              <Image src="../DefaultProfilePic.jpg" rounded/>
+                           </Col> */}
+                           <Col>
+                              <Card bg='dark' text='white'>
```
Comment (xhwallace) [inline src/components/pages/EditableProfilePage.js:31]: Make this card its own component to reduce the function size

File: src/components/pages/EditableProfilePage.js [hunk 3]
```diff
@@ -1,151 +1,179 @@
...
+                                    <Card.Text class="text-white">
+                                       {props.user.profile_info.bio} <BsPencil onClick={() => ActivateModal(["Bio", "bio", "profile_info"])}/>
+                                    </Card.Text>
+                                 </Card.Body>
+                              </Card>
+                           </Col>
+                           <Col></Col>
+                        </Row>
+                        <Row className="justify-content-md-center pb-3">
+                           {/* OLD Contact info box */}
```
Comment (xhwallace) [inline src/components/pages/EditableProfilePage.js:43]: Remove old info

File: src/components/pages/EditableProfilePage.js [hunk 4]
```diff
@@ -1,151 +1,179 @@
...
+                           {/* <Col xs={6} className="bg-dark text-white">
+                              <div className="h4">Contact Info</div>
+                              <div className="pt-">Email: {props.user.email} <BsPencil onClick={() => ActivateModal(["Email", "email"])} /></div>
+                              <div className="pt-4">Discord: {props.user.profile_info.discord} <BsPencil onClick={() => ActivateModal(["Discord", "discord", "profile_info"])} /></div>
+                              <div className="pt-4 pb-4">Steam Name: {props.user.profile_info.steam_name} 
+                                 <BsPencil onClick={() => ActivateModal(["Steam Name", "steam_name", "profile_info"])} /></div>
+                           </Col> */}
+                           <Card bg='dark' text='white'>
```
Comment (xhwallace) [inline src/components/pages/EditableProfilePage.js:51]: This card should also be its own component

**Review by xhwallace — APPROVED** [review]
Code is generally good. I suggest functionally decomposing the cards into their own components and removing the old contact info box info.

**Pull request merged**